src/Block/Service/AbstractBlockService.php

```diff
@@ -134,6 +134,12 @@ public function renderPrivateResponse($view, array $parameters = [], Response $r
 
     public function setDefaultSettings(OptionsResolverInterface $resolver)
     {
+        if (!$resolver instanceof OptionsResolver) {
```

Comment:
Really? I mean didn’t we keep it like this to allow the deprecated one? 🤔

Reply:
` configureSettings` needs an `OptionsResolver` instance, but we never checked this